imap/dav_db.c

```diff
@@ -193,13 +193,21 @@
     " PRIMARY KEY (rowid, ical_recurid, userid)"                             \
     " FOREIGN KEY (rowid, ical_recurid) REFERENCES jscal_objs (rowid, ical_recurid) ON DELETE CASCADE );"
 
+// dropped in version 16
 #define CMD_CREATE_CARDCACHE                                            \
     "CREATE TABLE IF NOT EXISTS vcard_jmapcache ("                      \
     " rowid INTEGER NOT NULL PRIMARY KEY,"                              \
     " jmapversion INTEGER NOT NULL,"                                    \
     " jmapdata TEXT NOT NULL,"                                          \
     " FOREIGN KEY (rowid) REFERENCES vcard_objs (rowid) ON DELETE CASCADE );"
 
+#define CMD_CREATE_JSCARDCACHE                                          \
```

Comment:
The equivalent `jscal_cache` table for calendar objects stores cache entries per userid. We could do the same thing here?